templates/base/generators/stores/index.js.hbs

```diff
@@ -0,0 +1 @@
+export { default } from './{{lowerCase name}}';
```

Comment:
This is not correct. 

it should be 

export { default } from './{{lowerCase name}}/contextProvider'

Reply:
this has been corrected